### PR TITLE
add-variables-descriptions

### PR DESCRIPTION
**Pull request by oanaBiancaSchiopu**

### CODE REVIEW

File: Triggers/resources/catalog/Memory_File_Trigger.xml [hunk 1]
```diff
@@ -3,12 +3,12 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xmlns="urn:proactive:jobdescriptor:3.13" xsi:schemaLocation="urn:proactive:jobdescriptor:3.13 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.13/schedulerjob.xsd"  name="Memory_File_Trigger"  priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2"  >
   <variables>
-    <variable name="HOST" value="" model="PA:MODEL_FROM_URL(${PA_SCHEDULER_REST_PUBLIC_URL}/rm/model/hosts)"/>
-    <variable name="FOLDER_TO_MONITOR" value="/tmp/trigger" model="PA:NOT_EMPTY_STRING"/>
-    <variable name="FILENAME_PATTERN_TO_MONITOR" value=".*" model="PA:NOT_EMPTY_STRING"/>
-    <variable name="CHANNEL_TRIGGERED_ID" value="FileTrigger_${PA_JOB_ID}" model="PA:NOT_EMPTY_STRING"/>
-    <variable name="CALLED_WORKFLOW" value="basic-examples/Print_File_Name" model="PA:CATALOG_OBJECT"/>
-    <variable name="FOUND_FILE_VARIABLE_NAME" value="file" model="PA:NOT_EMPTY_STRING"/>
+    <variable name="HOST" value="" model="PA:MODEL_FROM_URL(${PA_SCHEDULER_REST_PUBLIC_URL}/rm/model/hosts)" description="Hostname or IP address of the server"/>
```
Comment (fviale) [inline Triggers/resources/catalog/Memory_File_Trigger.xml:6]: Hostname or IP address of the server that will be monitored

File: Triggers/resources/catalog/Simple_File_Trigger.xml [hunk 2]
```diff
@@ -3,9 +3,9 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xmlns="urn:proactive:jobdescriptor:3.13" xsi:schemaLocation="urn:proactive:jobdescriptor:3.13 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.13/schedulerjob.xsd"  name="Simple_File_Trigger"  priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2"  >
   <variables>
-    <variable name="HOST" value="" model="PA:MODEL_FROM_URL(${PA_SCHEDULER_REST_PUBLIC_URL}/rm/model/hosts)"/>
-    <variable name="FOLDER_TO_MONITOR" value="/tmp/trigger" model="PA:NOT_EMPTY_STRING"/>
-    <variable name="FILENAME_PATTERN_TO_MONITOR" value=".*" model="PA:NOT_EMPTY_STRING"/>
+    <variable name="HOST" value="" model="PA:MODEL_FROM_URL(${PA_SCHEDULER_REST_PUBLIC_URL}/rm/model/hosts)" description="Hostname or IP address of the server"/>
```
Comment (fviale) [inline Triggers/resources/catalog/Simple_File_Trigger.xml:6]: Hostname or IP address of the server that will be monitored

File: Triggers/resources/catalog/Simple_File_Trigger.xml [hunk 3]
```diff
@@ -27,9 +27,9 @@
 It searches for files according to the pattern FILENAME_PATTERN_TO_MONITOR and is unblocked when at least one file is found. The task returns the list of file paths found as result. ]]>
       </description>
       <variables>
-        <variable name="HOST" value="" inherited="true" model="PA:MODEL_FROM_URL(${PA_SCHEDULER_REST_PUBLIC_URL}/rm/model/hosts)"/>
-        <variable name="FOLDER_TO_MONITOR" value="/tmp/trigger" inherited="true" model="PA:NOT_EMPTY_STRING"/>
-        <variable name="FILENAME_PATTERN_TO_MONITOR" value=".*" inherited="true" model="PA:NOT_EMPTY_STRING"/>
+        <variable name="HOST" value="" inherited="true" model="PA:MODEL_FROM_URL(${PA_SCHEDULER_REST_PUBLIC_URL}/rm/model/hosts)" description="Hostname or IP address of the server"/>
```
Comment (fviale) [inline Triggers/resources/catalog/Simple_File_Trigger.xml:30]: same remark

**Review by fviale — APPROVED** [review]
LGTM, minor comment

**Review by aminelouati — APPROVED** [review]

File: Triggers/resources/catalog/Memory_File_Trigger.xml [hunk 4]
```diff
@@ -3,12 +3,12 @@
...
   <variables>
-    <variable name="HOST" value="" model="PA:MODEL_FROM_URL(${PA_SCHEDULER_REST_PUBLIC_URL}/rm/model/hosts)"/>
-    <variable name="FOLDER_TO_MONITOR" value="/tmp/trigger" model="PA:NOT_EMPTY_STRING"/>
-    <variable name="FILENAME_PATTERN_TO_MONITOR" value=".*" model="PA:NOT_EMPTY_STRING"/>
-    <variable name="CHANNEL_TRIGGERED_ID" value="FileTrigger_${PA_JOB_ID}" model="PA:NOT_EMPTY_STRING"/>
-    <variable name="CALLED_WORKFLOW" value="basic-examples/Print_File_Name" model="PA:CATALOG_OBJECT"/>
-    <variable name="FOUND_FILE_VARIABLE_NAME" value="file" model="PA:NOT_EMPTY_STRING"/>
+    <variable name="HOST" value="" model="PA:MODEL_FROM_URL(${PA_SCHEDULER_REST_PUBLIC_URL}/rm/model/hosts)" description="Hostname or IP address of the server that will be monitored"/>
+    <variable name="FOLDER_TO_MONITOR" value="/tmp/trigger" model="PA:NOT_EMPTY_STRING" description="Folder on the given HOST that will be monitored for new files presence"/>
+    <variable name="FILENAME_PATTERN_TO_MONITOR" value=".*" model="PA:NOT_EMPTY_STRING" description="Only filenames which match the supplied regular expression will be considered"/>
```
Comment (nebilbenmabrouk) [inline Triggers/resources/catalog/Memory_File_Trigger.xml:8]: Only filenames which match the supplied regular expression **are** considered

File: Triggers/resources/catalog/Memory_File_Trigger.xml [hunk 5]
```diff
@@ -3,12 +3,12 @@
...
-    <variable name="FILENAME_PATTERN_TO_MONITOR" value=".*" model="PA:NOT_EMPTY_STRING"/>
-    <variable name="CHANNEL_TRIGGERED_ID" value="FileTrigger_${PA_JOB_ID}" model="PA:NOT_EMPTY_STRING"/>
-    <variable name="CALLED_WORKFLOW" value="basic-examples/Print_File_Name" model="PA:CATALOG_OBJECT"/>
-    <variable name="FOUND_FILE_VARIABLE_NAME" value="file" model="PA:NOT_EMPTY_STRING"/>
+    <variable name="HOST" value="" model="PA:MODEL_FROM_URL(${PA_SCHEDULER_REST_PUBLIC_URL}/rm/model/hosts)" description="Hostname or IP address of the server to be monitored"/>
+    <variable name="FOLDER_TO_MONITOR" value="/tmp/trigger" model="PA:NOT_EMPTY_STRING" description="Folder on the given HOST that will be monitored for new files presence"/>
+    <variable name="FILENAME_PATTERN_TO_MONITOR" value=".*" model="PA:NOT_EMPTY_STRING" description="Only filenames which match the supplied regular expression are be considered"/>
+    <variable name="CHANNEL_TRIGGERED_ID" value="FileTrigger_${PA_JOB_ID}" model="PA:NOT_EMPTY_STRING" advanced="true" description="Name of the channel in synchronization api that will be used by this workflow"/>
```
Comment (nebilbenmabrouk) [inline Triggers/resources/catalog/Memory_File_Trigger.xml:9]: Same for the other variables @oanaBiancaSchiopu 
For example,  "Name of the channel in synchronization api **to** be used by this workflow"
Thanks

**Pull request merged**